### PR TITLE
LPS-147612 | Translation results displayed side by side

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
Create automation tests for Translation Manager

**Test Plan**

Create a page with the widget JS Components Sample
Click AUI Tag translation selector of Admin mode
Input different languages with the same word
View translation results displayed side by side

**JIRA Ticket:**
https://issues.liferay.com/browse/LPS-147612

### CODE REVIEW

File: modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+			AssertElementPresent(locator1 = "Button#MANAGE_TRANSLATIONS");
+		}
+
+		task ("Assert link to Manage Translations is available") {
+			Click(locator1 = "Button#MANAGE_TRANSLATIONS");
+
+			AssertElementPresent(locator1 = "Portlet#MODAL_TITLE");
+		}
+	}
```
Comment (john-co) [inline modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:81]: Looks like leftover testcase for the other PR in https://github.com/liferay-frontend/liferay-portal/pull/1929

File: modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			AssertElementPresent(
+				key_title = "Admin",
+				locator1 = "TextInput#INPUT_TEXT_FIELD",
+				value1 = "Bienvenue sur Liferay");
+
+			AssertElementPresent(
+				key_title = "Admin",
+				locator1 = "TextInput#INPUT_FORM_TEXT",
+				value1 = "Welcome to Liferay");
+		}
```
Comment (john-co) [inline modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:105]: Sorry, I don't quite understand what we're trying to test here. It looks like we're just inputting text into a couple forms and asserting that they're present? I think maybe `editTranslationInput` and the task `Input different languages with the same word` and  description `Verifies translation results displayed side by side.` Don't tell me enough to understand what's going on. I tried to read the Given When Then in https://issues.liferay.com/browse/LPS-147612 but it also doesn't make sense to me... Given When Given sequence doesn't make sense.

Could someone explain or show me a video of what we're trying to test? cc/ @zhangyan0701 @Hijky

Comment (john-co) [reply modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:105]: I reviewed the original test scenario which involves web content. This is what I found which looks different than what's available on js components sample portlet. To me, this is not testable with current js components sample portlet.
image

https://user-images.githubusercontent.com/35235266/155219370-d8654186-a519-4f06-b1cd-96ff0cc53bb5.mp4

cc/ @edalgrin @tinycarol @dsanz, is the side by side UI/feature something we will support for Translation Manager for all product teams? imo, it looks like an implementation only with Web Content and is outside the scope of testing for Translation Manager on FI side.

**Comment by john-co** [comment]
Closing this, changes should be applied to the new PR: https://github.com/liferay-frontend/liferay-portal/pull/1935

File: modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase (hunk 2 above)
Comment (dsanz) [reply modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:105]: The side-by-side is out of the scope of this module. That's a very particular use case where a variant of translation manager which does not sync language selections could be used, but we're not there yet.

Comment (john-co) [reply modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:105]: Thanks @dsanz ! 

@Hijky , please remove this test case from the in progress PR.

Comment (Hijky) [reply modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase:105]: ok, I will remove it.

**Pull request closed without merging**